src/Campaigns/resources/admin/components/CampaignDetailsPage/style.module.scss

```diff
@@ -199,14 +199,115 @@
             margin: 0 0 var(--givewp-spacing-2);
         }
     }
+
+    .sections {
```

Comment:
We need to rename this file and the places that are using it to `CampaignDetailsPage.module.scss` because the way it's being used right now is preventing the CSS files from the Form List table from being loaded - check the attached screenshot.

![image](https://github.com/user-attachments/assets/de0ac574-7f7e-4fef-91d1-0817a9965af9)

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7547/commits/8068f9b8474f1df1e36e0eb9a6fac7dad23876d1